README.md

```diff
@@ -1,10 +1,18 @@
 # node-red-contrib-influxdb
 
-A <a href="http://nodered.org" target="_new">Node-RED</a> node to write and query data from an influxdb time series database.  These nodes use the <a href="https://www.npmjs.com/package/influx" target="_new">influxDB 1.x client</a> for node.js, specifically calling the **writePoints()**, and **query()** methods.  Currently these nodes can only communicate with one influxdb host.  These nodes do not support InfluxDb 2.0 yet.
+<a href="http://nodered.org" target="_new">Node-RED</a> nodes to write and query data from an InfluxDB time series database. 
+
+In a first operation mode, selectable with a como box, these nodes use the <a href="https://www.npmjs.com/package/influx" target="_new">influxDB 1.x client</a> for node.js, specifically calling the **writePoints()**, and **query()** methods. Currently they can only communicate with one influxdb host. These nodes are used for writing and querying data in InfluxDB 1.x to 1.8+.
```

Comment:
It's not really an 'operation mode', but a 'version selector'.  I think the versions be 1.x, 1.8-flux, 2.0, not 1.8, 1.8-flux, 2.0

Reply:
Sounds good